posthog/batch_exports/http.py

```diff
@@ -147,6 +149,17 @@ def retry(self, *args, **kwargs) -> response.Response:
             batch_export_run.data_interval_end,
         )
 
+        log_activity(
+            organization_id=cast(UUIDT, self.organization_id),
+            team_id=self.team_id,
+            user=request.user,
+            was_impersonated=is_impersonated_session(request),
+            scope="BatchExportRun",
+            item_id=str(batch_export_run.id),
+            activity="retry",
```

Comment:
I think you'd have to add this as an activity iirc
might be better to be created (since we're creating the retry)
and then the detail says it's a retry?

not a strong opinion it's free to add new activity types 🤣